.vscode/cmake-kits.json

```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "name": "InfiniTime Compiler",
+        "environmentSetupScript": "${workspaceFolder}/docker/build.sh",
+        "compilers": {
+            "CC": "/usr/bin/gcc",
+            "CXX": "/usr/bin/g++"
+        }
+    }
+]
```

Comment:
What's the purpose of this file?

Reply:
This defines the standard compiler used inside the devcontainer. It is a mistake though that the compiler is set to /usr/bin/gcc, that is the standard compiler included in ubuntu.